satellite/satellitedb/accounting.go

```diff
@@ -20,6 +23,71 @@ type accountingDB struct {
 	db *dbx.DB
 }
 
+// ProjectBandwidthTotal returns the sum of GET bandwidth usage for a projectID for a time frame
+func (db *accountingDB) ProjectBandwidthTotal(ctx context.Context, bucketID []byte, from time.Time) (int64, error) {
```

Comment:
This method should have only `projectID` as parameter.

Reply:
the test kept failing for this because of the way i was using projectID so i changed it to bucketID to parse it like how we do in `UpdateBucketBandwidthSettle`.  Could we leave this alone for now until we rethink passing the all the args as `bucketName, projectID`?